### PR TITLE
docs: added a warning and a info panel in the Quickstart Installation page

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

File: docs/main/quickstart-guide/installation.md [hunk 1]
```diff
@@ -43,6 +43,21 @@ At a high-level, OpenEBS requires:
 
 ## Installation via Helm
 
+:::warning
+**Deprecation Notice: Helm Chart Registry Migration**
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
```
Comment (avishnu) [inline docs/main/quickstart-guide/installation.md:49]: This and the below 2 lines give conflicting messages

File: docs/versioned_docs/version-4.0.x/quickstart-guide/installation.md [hunk 2]
```diff
@@ -43,6 +43,21 @@ At a high-level, OpenEBS requires:
 
 ## Installation via Helm
 
+:::warning
+**Deprecation Notice: Helm Chart Registry Migration**
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
```
Comment (avishnu) [inline docs/versioned_docs/version-4.0.x/quickstart-guide/installation.md:49]: Same comment as above.

File: docs/main/quickstart-guide/installation.md (hunk 1 above)
Comment (balaharish7) [reply docs/main/quickstart-guide/installation.md:49]: @avishnu Then, can we remove the "**Deprecation Notice** Warning" & keep only the "**Important** panel" (i.e. in line 70)

Comment (avishnu) [reply docs/main/quickstart-guide/installation.md:49]: I think the text message needs to be modified. Something like:

The Helm chart registry at "https://openebs.github.io/charts" has now been deprecated as those charts are used to install legacy OpenEBS (v3.10 and below) releases. This registry will be migrated to a different registry location "https://openebs-archive.github.io/charts" by Oct 30, 2024.

The Helm charts for the latest OpenEBS (v4.0 and above) are hosted in a new registry location "https://openebs.github.io/openebs". To ensure seamless access to OpenEBS Helm charts, update your configurations to use the new registry URL.

File: docs/main/quickstart-guide/installation.md [hunk 3]
```diff
@@ -52,6 +67,10 @@ helm repo add openebs https://openebs.github.io/openebs
 helm repo update
 ```
 
+:::important
+`helm repo add openebs https://openebs.github.io/charts` is changed to `helm repo add openebs https://openebs.github.io/openebs`.
```
Comment (niladrih) [inline docs/main/quickstart-guide/installation.md:71]:
```suggestion
`helm repo add openebs https://openebs.github.io/charts` has changed to `helm repo add openebs https://openebs.github.io/openebs`.
```

**Review by avishnu — APPROVED** [review]

**Pull request merged**